pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>com.google.auth</groupId>
   <artifactId>google-auth-library-parent</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
```

Comment:
This is the right version change. It also needs to be updated at the top of the individual sub-component pom.xml files also. We have explored this before, and there is not a known way to share this value in a single field accross the poms.